packages/augur-artifacts/src/upload-block-numbers.json

```diff
@@ -1,13 +1,13 @@
 {
-  "1": 5926223,
-  "3": 3718180,
-  "4": 4663954,
-  "19": 1340162,
-  "42": 7926279,
-  "101": 113,
-  "102": 4,
-  "103": 3,
-  "104": 1,
-  "8995": 0,
-  "12346": 123
+ "1": 5926223,
```

Comment:
Or the upload block changes

Reply:
These are the values on master, no idea why they came up in this PR. I merged with master and double checked. The values are the same.